stake-o-matic/testnet-validator-whitelist.yml

```diff
@@ -0,0 +1,140 @@
+- AiWqv1dqsbvkUMec7G4DmM88ka7SaoqDPkn5U2iuvqnn
+- AicKhNhJmkdqafRDjKLPgVqLzXLzJ8pS6aVrYrRkq1iq
+- AkVMbJq8pqKEe87uFaxjmt35tX2cNhUJTJwv13iioHu7
+- AkkJv1meyo2Ax2XTXEXWpvHTh4F8a68Lja5dx3TaX47K
+- Aom2EwxRjtcCZBDwqvaZiEZDPgmw4AsPQGVrsLa2srCg
+- B4xFUYq2TDmz6PsiCj29vFyvmYvqTRAtnhQ3uSmxQVFd
+- B52Da5MCyTcyVJEsR9RUnbf715YuBAJMxCEEPzyZXgvY
+- B6ZramCQhQWcq4Vxo3H5y1MYvo37wRZiAwKk88qaYNiF
+- B8T2dbvYaM4gJfpLLbuBScuWKFhnG6KspRK72v5D5uoK
+- BLfpk2WoF8RnerCqjxHe7qFJjpEE427riMkf2fS6vUB6
+- BSF2yD9mqzaixDaLEraF1en82EWaXx7wbaCqSuKppqG5
+- BWiftESMUsve87rkjU7HsaA7fkiJRAbv3xZLQrmKZtnz
+- BhbARoxdh2MT3vb4awXraZFPzSwBdmF9pGgURKNsjBqC
+- C4N1bMSzbfDwHGMitxyufNZPaAkNYx8vJxHRnHWrptAT
+- CPPVEbGFbX3XAThetvfveCE1vYLWUwwJGT7DxkPAWb8D
+- CV3F19YAhoW7DpfHQ5W9t2Zomb9h21NRi8k6hCA36Sk6
+- CZWpCTN4rCWer8fm5ZqFdx82CDiCJjZLKZ5Ti2gdmchQ
+- ChorusM5BVgnAKbg9PF15285LkqeCoZWK2p9s35T7J2A
+- CiY5RjWPs1XyegKyBLcG7Ue7YMf98eiEnmvqnSuSKbob
+- CsaAGRau3ZvyMQvJ9CWSqbqeVv9zw2Am8FhnL9sr6jTk
+- D23NCAVxinE53BTemguZCheAqCdMGfNTUzWdoWvq4Xj
+- D2NjDkcv8Y1dWGdtWAKPT4em2D3sYzM8AzMTpCG1RVf7
+- D52Q6Ap8RVMw1EvJYTdEABP6M5SPg98aToMcqw7KVLD9
+- D71JRzjPpHipt8NAWnWb3yZoXezbkGXqSf7TVCir6wvT
+- DFb6qaAkd5DTnFVYLDjzJNfsUPygP8GHHebN1CBv25cf
+- DJvMQcb3ZtXC49LsaMvAo4x1rzCxjNfBfZtvkUeR4mAx
+- DKnZytVA5wKbNPYW1pvPpoE5YeSsxu12KJFa95gBAGm7
+- DaB3ZwVtGLzSjazk5STQEu3MkJR2nkK3tDdCPAvx9QpM
+- DadnDZbFH5BHHRHD7TaobaSQ7QATXgvWegHUcZ7ZGzmW
+- DadnDZbFH5BHHRHD7TaobaSQ7QATXgvWegHUcZ7ZGzmW
+- Db2V7nPHc4sPHne87nYXPGn8Kv8rMsiWCAjgAXmpqcpC
+- DsnqNtwKA817a2VQypWEzaRXY2soq5Jgc68MgFBMR35p
+- Dx4bMuKpGaxAnd53QYDyKhD45PjuFLx16mrgoRK36STf
+- DzxNmWD99qvkPdDR94ojXhUrpzT8VdqB5ktYX7fZr4gc
+- EBxhSfAWW2Cfouvj1k242W6U8krZVAxJS47SG8UKb4ch
+- EWg9NTC5s7Pa9FktUk6dX8xRYkvJ952peH1z1iznd4nV
+- ErbvzZx2Ss9GxizKyDviybhZPu8noHv4AM5vuzTh1ij6
+- F7FgS6rrWckgC5X4cP5WtRRp3U1u12nnuTRXbWYaKn1u
+- FLjMwfY9ahmyc6VgQpokBrra5Ucf5KSK5YW5x9MMKnpS
+- Fb5cEcYNgPXKJoEmvPvsU2ENYRVePQtExqgf77AnVX54
+- Fou7Du6KtVb8dVMzKMYW39fuSGpMzJGwpkQ45NbxA3Tx
+- Ft5fbkqNa76vnsjYNwjDZUXoTWpP7VYm3mtsaQckQADN
+- Fv4zJ7RvV8gEYxEtLjnGZAX1qxjqRh56DzBgqvFEVjjM
+- FzAv1TFpCyR65GrxeqBwnEzNVXEeUMPV5rKZGQhPR7mq
+- G7NoACjYzRcrLibw3dbNoDCVoBR67ijm3tYeL5zL5qKY
+- GQnaJJu7h53SVVhVpg2ErkSKhYMtqYrqv1qr13MUobuq
+- GbtVg3D6bNFSjem21vrJBJTpUniwwEtmvs8mQkX5XS1V
+- GuKn8nEJwUPjBfxpwyq2MXU2JNrSpj7gqnKptCZeEk7j
+- H4Dgb3KyCuYWKT8yKtp8qbY49cvaqZcisa2GDnroFsv6
+- HGHMEEHCfbVFjqB69Hu9oNW6SviukB8jUheEhYVZJKe2
+- HM2hzFLTd5TAhejGFjaXAm8LLjdmnj7bqQrzpRTaawdo
+- HUoud6qywaWj8kZwdHRTbEPkKmskHa6Md1KNvF1JQFYF
+- HgmPwzNcY85HfrN3bYiqaypb6Nmf7ayaZEaivGY37913
+- HwFvyMbGLkiTUaT66cfL1FnJ26c9VqtpqAg4UbWSXtdq
+- URnkWZGiuB7jXbfCSuNSwir1qkn7sXjiSPeLPaXys7b
+- VzZqDXSDF8KbfAQUvizj52NTjRg9cfXCAZ1cUJt5614
+- YYYYW8eKkmwQFhVGUKdBAnDQPuhMTpG7zwm9nikNndC
+- aeypRbHck9k5zGuftrdSaUMfjJyTrjtaGJEtALt835y
+- fish2tcDFRiAM1cNKqF7b96WzQmmDRjsf5XD214MhpV
+- h4idQXKHx1SBgzXeYAsuDAFvLzyepV2yr1o9wxKsGdx
+- hQBS6cu8RHkXcCzE6N8mQxhgrtbNy4kivoRjTMzF2cA
+- xViooFwobBZE416moj2yFV6RJwGVaBDbCQJRfCyz2FS
```

Comment:
This is the balance of keybase-based pubkeys that aren't whitelisted.  A bunch of the recent discord complaints of not getting whitelisted came from people who registered via the keybase form a month ago, but are just getting started now.  If it won't hurt the stake-o-matic, I'd like to pre-empt further complaints and add all of the addresses that have come in from keybase that aren't currently on the whitelist.

```suggestion
- xViooFwobBZE416moj2yFV6RJwGVaBDbCQJRfCyz2FS
- 8ASvX43WvDF4LHe2ioNUbn3A4ZRD9iC6rsxyjo5sVGLp
- k4wTUPNkKM6b3LDSh3KbXsMCj64LoGmMB4W4iw2shXE
- Csrv9JCbebTKu1uBWqkfwuPHwVCXsYDrQmeXf19onbsY
- EU7Kd1fhYgUP21nSkev4DfYJQc2PmFGRXBDXRL1pMyGK
- FRgKM5ZXu9e3U7RrPqKiksm4v7MWcy14EixGQKufd8oG
- DciwdVV1DXimdsgRGQuQ45zYVjZNaof6a6EZ1JjaCsvx
- 74U9LiSPv2gb8bBZSh5uVNf89W4wZ8zs9B8EvRVVwr87
- 59WHuha1QunWmupWhFA4vr3WMUe8BLN7dc8HUsJ4YC86
- FzMsVyZXVuMRCwirhnb12MS64JPdtVY5R3jCwQ6C5JpX
- 86hxQGfre1vvVsYP7jPBka8ySrECZQewjG3MfxoCoBYf
- 4YGgmwyqztpJeAi3pzHQ4Gf9cWrMHCjZaWeWoCK6zz6X
- 58J9ucd9Qc6gMD8QHh2sHTyJyD8kdjHRQZkEAyAZ72YA
- FQigEZ2cSQZPLNjK5NMbt7oquyg17Q27cqQBMEJr4dQf
- BV8sS1jn1AvGAptY5TxNZdcm7aa49MZCXSpXQjzjdnYG
- AmNRXJSSaGJrXaYBahD1PBoBR2ApkfLBAJ7SiRK4772R
- 9fMqL641B7nQZ1xktU35qXuFESxMB7pkqmqVANtovzmE
- 31cud34DHkL5zM4ZiHXgsrkAu13Jeck7ahvkPU9i4Jze
- 9waRyqWAoP68etU17DdWamgpTnPb3skY7Er9kRZMzCfS
- 2jYzt9Ly7dNzNpMV9sbHBNuwMEvVdSi9L8yJCTJT21ki
- 9mbQ9mrRjBGiUVnz9Tdf7PuAkWomw3GLZ6Sup3R93Gw8
- 7FnrBgjPb1y8PNjzRLihQWUvky37F7wkvRb7MUL89Q8P
- HS9Kg3kHiyNCgDLHLv62DThvqaqQUiptWkbW3PmTXxLk
- 5dLMRyPWx6rdPGZpZ7uuZiqry96dUT5yz48u62Gzugi6
- 2ZeChc7Res7fUVdcGCDTJfRd9N8R21hiBPLAuJsqHHwh
- AoUwfPuiEek2thVRDhMP7HbQb9rguyab4rDiz2NAfwwA
- 6t8zWy766tsHBVNxhMwsTGiEYkGtjaZncRU3vcSEYtHU
- 8FaFEcUFgvJns6RAU4dso3aTm2qfzZMt2xXtSgCh3kn9
- G9nU3uDUXkH5opUt4niAxv3qh5ojFAZLLfvQEMv4EFVJ
- 7yzVecfpWupdJwVQby3inMggGSotFSnSrhvPGPp6JGdU
- F48fyuAYvfJgY4eTXkxmJwwXP6jFVFmQkzt4N4TSiGGF
- 4vgoKb76Z2vj9V9z7hoQpZkkwJrkL1z35LWNd9EXSi2o
- 75yzn7njq32yY1ieCZxFNVFZQWtEbQHpaaG6dSZFfmX5
- 8XvoJswfYCxWf2WkUmNBjtDWwonmKugYhxBruNpSfS4a
- CcorN3BoG1XMZehZ9Xib9YLo4mcvo7pzeVurC28gYYqX
- CtxU5HwVbgspJVtWxwjuP8wXUMdkjYJ4EJwJ3jvZh4zu
- DV78gathrorcpWsWrUkWrWNowLXpizKsPBupStzeAJnL
- HvSBzPhtU8RqjmyqFTVZtM4seYHZisz11G2MR5ZDxzJN
- 5MGfrpdVsyifhn2x62j6PnBWQk2c5xTUK1o8KFrwPLWG
- 6TkKqq15wXjqEjNg9zqTKADwuVATR9dW3rkNnsYme1ea
- ES5G93qiEu5hCyMZMpFUhnbK4YZKhrZMN86Xfkub9YJL
- D5JqF3qkLkeJKKEi145oMseEGc1ym9cWKtBKtg4ZBBnN
- 5PLDu8auwqtMkHW9zdPsfUcvyESZ45umFc4r8cWUw3Zp
- Drkj3wbHHmE2iCnqXHKFTmwPkuSc4bsFdgAmqv6eXuWi
- 23SUe5fzmLws1M58AnGnvnUBRUKJmzCpnFQwv4M4b9Er
- 6zCt5z72rfN9sRk2hTgc1LeFDbEBfXYmW6xtSNmgyama
- HrKX44KVBJByNgaMBkgweKipFvQchGGkhixY6kNtKLDD
- BMZY98zbjg2ey4XNfhBQXhEuvVqzaJ1T3AKD2quL3wnK
- B5FNFrfrrfpBggFBp9h6Js93mrz1Tham1z1GgP3bDgNc
- 3pzTi41c6NAbZyTiyPEAQtmi2K5MyWZJMxx6nDvWPgnQ
- 4FZSiJpGgprsVxkzc2F8v3bgnRpk8Ez1Dq7ohXwY1q9V
- 2YLPihCDxqztR5be69jhoNDPMxV6KeTJ2X2LtVBXDgp4
- FiEEzxMnedqDnDRStwshHNQTrD9DykkCSmbMcreqQuCW
- 5uTcsQSrUffYo6RYSWj75SuGMkJ4v9x5RYuQoTc5aWGR
- 6nrkRvzUpTst8teZJawMFFHrmixJ2sxAUxPKrqoGwCB8
- 8uJR7fEaTiriYLe4YMFkeeADdRkexxm8jkFCGRjPBvJD
- CjhKCjNC1WUgBjAGst3D2XmSsWHvt3zGasasmogPTY6J
- Ag51oCkwGy5rkbGEYrcP9GDjvFGMJrEdLxvedLSTSR15
- HoMBSLMokd6BUVDT4iGw21Tnxvp2G49MApewzGJr4rfe
- 4WufhXsUhPc7cdHXYxxDrYZVVLKa9jCDGC4ccfmuBvu2
- 7yiFPLeyH2B2p3xGrr2Y5nmk8B52nEaa6j55Nk3u6648
- 3ckQZncmgmS1aZCC7Eot659hoBQ8k85ExddD7fu9qJ2c
- 2XAHomUvH3LFjYSxzSfcbwS73JgynpQHfapMNMJ8isL9
- 5qsT9h2TuTMPLtX6gcD2DG6mcZwechxmWhKRFHbCt6Pu
- 787PK2WaCUZCyYEmuYQSGmoxu7MyqK1usn43FfiVwhcB
- 3xUTkgPKNJZ3dkpDMV8zWV34BkmvKanguKipv6M9x2Mt
- 3wz211BhQAE2n5fjDQSStM2iSizhNRyJDNRkDEc1YwMF
- 5sjXEuFCerACmhdyhSmxGLD7TfvmXcg2XnPQP2o25kYT
- Co7UqfqzXzTjhBwvam3zhNi4p8dKtdSrfh6rQykoNMy7
- 368KipD4nBzVs4AizHj1iU4TErSSqmZaNGVyzHx8TVXM
- 9a95Qsywtw2LKK9GkiR9dpTnoPGMVrLBLTAMPnaUwDjY
- 8nvD1CUE48WdcmRdvbyWcM5LdJKRTNP3tXT6Qp2CSND5
- ACv5dTk7THbmUpHYGhgPzMhWr7oqHSkuPJpa5RfvmG5H
- DfTeDaxk4RufkVbykedVnqa1r9S3z3oKFYL3FFmPdr1o
- 3HitRjngqhAgVuNdFwtR1Lp5tQavbJri8MvKUq5Jpw1N
- 2CGskjnksG9YwAFMJkPDwsKx1iRAXJSzfpAxyoWzGj6M
- CM1c6z3pRNgHFcfZG4z3wE31jaR8c4gCYQBJVEoCUyq8
- 4gMboaRFTTxQ6iPoH3NmxLw6Ux3SEAGkQjfrBT1suDZd
- FNdoUuKVBigMFGpVvSMLXJB4FC7XQL1RjPUqUiwvPiCS
- FhPGvLZRVn97AsZ5grffU7qhaRXrmxD4WcJgbLYkJh1K
- 2tZoLFgcbeW8Howq8QMRnExvuwHFUeEnx9ZhHq2qX77E
- Dq5r3zG6XGBcXNDRSWPSc7DiWwjqcGoiVcEhZ9mXEAaV
- 4vXPjSaZfydRqhnM85uFqDWqYcFyA744R2tjZQN8Nff4
- 4ZtE2XX6oQThPpdjwKXVMphTTZctbWwYxmcCV6xR11RT
```

Reply:
The only harm is more baseline/bonus stake account created so need more SOL, and I only have enough TdS SOL for 24 more accounts right now

Reply:
Baseline stake is only for non-delinquent validators right?  So if some of these addresses are offline, then no harm.  If they are online and not whitelisted, we should work to get them staked along with everyone else, right?

Reply:
No som creates the baseline/bonus stake accounts upfront for everybody in the whitelist